### PR TITLE
Use nice icon for wait_serial

### PR DESCRIPTION
**Pull request by asdil12**

### CODE REVIEW

**Comment by okurz** [comment]
Could you show us how it looks like in both cases where you added it?

File: templates/step/moduleslistthumbnails.html.ep [hunk 1]
```diff
@@ -15,7 +15,11 @@
                     % end
                 % } elsif ($refimg->{text}) {
                     %= link_to url_for('step', stepid => $img_count) => (class => 'no_hover') => title => $refimg->{title} => begin
-                        <span class="<%= "resborder_\L$refimg->{result}" %>" style="width:<%= $ref_width %>px;height:<%= $ref_height %>px;" ><%= $refimg->{title} || 'Text' %></span>
+                        % if($refimg->{title} == "wait_serial") {
+                            %= image '/images/terminal.svg', alt => $refimg->{name}, width => $ref_width, height => $ref_height, class=> "resborder_\L$refimg->{result}"
```
Comment (aaannz) [inline templates/step/moduleslistthumbnails.html.ep]: Nitpick: missing space in `class=>`

File: templates/test/result.html.ep [hunk 2]
```diff
@@ -172,7 +172,11 @@
                                                     % end
                                                 % } elsif ($step->{text}) {
                                                     %= link_to url_for('step', moduleid => $module->{name}, stepid => $step->{num}) => class => 'no_hover' => title => $step->{title} => begin
-                                                        <span class="<%= "resborder_\L$step->{result}" %>" ><%= $step->{title} || 'Text' %></span>
+                                                        % if($step->{title} == "wait_serial") {
+                                                            <img src="/images/terminal.svg" width="60" height="45" alt="<%= $step->{name} %>" class="<%= "resborder_\L$step->{result}" %>"/>
```
Comment (aaannz) [inline templates/test/result.html.ep]: In the diff above you used `%= image ...`, here you are using `<img src=....`. Why so?

Comment (asdil12) [inline templates/test/result.html.ep]: Because that was used for audio.svg in each file.

**Comment by coolo** [comment]
can you try with a font awesome icon instead?

**Comment by asdil12** [comment]
@coolo I tried, but sizing was broken.

**Comment by coolo** [comment]
yeah, had thought a second after I posted - it should be possible to embed one in a bigger box. But not worth the effort

**Comment by okurz** [comment]
+1

File: templates/step/moduleslistthumbnails.html.ep [hunk 3]
```diff
@@ -11,11 +11,15 @@
 		    %= step_thumbnail($refimg, $ref_width, $testid, $moduleid, $img_count)
                 % } elsif ($refimg->{audio}) {
                     %= link_to url_for('step', stepid => $img_count) => (class => 'no_hover') => begin
-                        %= image '/images/audio.svg', alt => $refimg->{audio}, title => $refimg->{audio}, width => $ref_width, height => $ref_height, class=> "resborder_\L$refimg->{result}"
+                        %= image '/images/audio.svg', alt => $refimg->{audio}, title => $refimg->{audio}, width => $ref_width, height => $ref_height, class => "resborder_\L$refimg->{result}"
                     % end
                 % } elsif ($refimg->{text}) {
                     %= link_to url_for('step', stepid => $img_count) => (class => 'no_hover') => title => $refimg->{title} => begin
-                        <span class="<%= "resborder_\L$refimg->{result}" %>" style="width:<%= $ref_width %>px;height:<%= $ref_height %>px;" ><%= $refimg->{title} || 'Text' %></span>
+                        % if($refimg->{title} == "wait_serial") {
```
Comment (coolo) [inline templates/step/moduleslistthumbnails.html.ep]: damn, I overlooked that you compare strings with == here

**Pull request merged**